civis/parallel.py

```diff
@@ -761,18 +775,24 @@ class _CivisBackend(ParallelBackendBase):
 
     Users should interact with this through ``make_backend_factory``.
     """
+    uses_threads = False
```

Comment:
Is setting this to `False` unnecessary?  I'm fine with keeping it, to be explicit.

(It looks like the other backends [don't always set all three of these](https://github.com/joblib/joblib/blob/acca6b95ed9e28498441d630ed2ddf674b66c8b8/joblib/_parallel_backends.py#L171-L172))

Reply:
I think it's unnecessary, yes. I added it to be explicit. The places in the joblib code which look for it access it via a `getattr` that defaults to `False`.